vaccine/states.py

```diff
@@ -254,3 +265,39 @@ async def display(self, message: Message):
                 ],
             },
         )
+
+
+class SectionedChoiceState(ChoiceState):
+    def __init__(
+        self,
+        *args,
+        sections: Sequence[Tuple[str, Sequence[Choice]]],
+        separator: Optional[str] = None,
+        **kwargs,
+    ):
+        choices: List[Choice] = []
+        for section_choices in sections:
```

Comment:
Not a big one, but could probably do `for _, choices in sections` instead of the `section_choices[1]` below